minecraft_servers/crystopia/manifest.json

```diff
@@ -0,0 +1,40 @@
+{
+    "server_name": "crystopia",
+    "nice_name": "Crystopia",
+    "direct_ip": "crystopia.net",
+    "supported_languages": [
+       "en",
+       "de"
+    ],
+    "social": {
+      "web": "https://www.crystopia.net",
+      "web_shop": "https://shop.crystopia.net",
+      "web_support": "https://www.crystopia.net/wiki",
+      "twitter": "crystopianet",
+      "discord": "https://discord.com/invite/q7xW3MH425",
+      "youtube": "https://www.youtube.com/@crystopia"
+    },
+    "gamemodes": {
+      "smp": {
+        "name": "SMP", 
+        "color": "#FFA500",
+        "url": "https://www.crystopia.net/games/smp",
```

Comment:
Returns a 404

Reply:
Yes i fix it later page Display some objekts not correct because its disabeled!